src/main/java/uk/gov/hmcts/sptribs/cdam/model/Document.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.sptribs.cdam.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Setter;
+
+import java.util.Date;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Setter
+public class Document {
```

Comment:
Is it possible to give a different name as Document class is already available in ccd-config-generator and is used widely for all our documents related to case. Though having the same name doesn't fail compilation but a bit of confusion. Happy to discuss if needed,